<ui version="4.0" >
 <class>OsdConfig</class>
 <widget class="QWidget" name="OsdConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>472</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="kcfg_OsdEnabled" >
     <property name="toolTip" >
      <string>Check to enable the On-Screen-Display. &lt;br>The OSD briefly displays track data when a new track is played.</string>
     </property>
     <property name="whatsThis" >
      <string>Check to enable the On-Screen-Display. The OSD briefly displays track data when a new track is played.</string>
     </property>
     <property name="text" >
      <string>&amp;Use On-Screen-Display</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Q3GroupBox" name="mainBox" >
       <property name="title" >
        <string/>
       </property>
       <property name="frameShape" >
        <enum>Q3GroupBox::NoFrame</enum>
       </property>
       <layout class="QVBoxLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="Q3GroupBox" name="fontBox" >
          <property name="title" >
           <string>&amp;Font</string>
          </property>
          <property name="flat" >
           <bool>false</bool>
          </property>
          <property name="frameShape" >
           <enum>Q3GroupBox::GroupBoxPanel</enum>
          </property>
          <property name="frameShadow" >
           <enum>Q3GroupBox::Sunken</enum>
          </property>
          <property name="lineWidth" >
           <number>1</number>
          </property>
          <layout class="QVBoxLayout" >
           <item>
            <widget class="QCheckBox" name="kcfg_OsdDrawShadow" >
             <property name="text" >
              <string>Draw &amp;shadow</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Q3GroupBox" name="groupBox9" >
          <property name="title" >
           <string>C&amp;olors</string>
          </property>
          <layout class="QVBoxLayout" >
           <item>
            <layout class="QVBoxLayout" >
             <property name="spacing" >
              <number>2</number>
             </property>
             <item>
              <widget class="QCheckBox" name="kcfg_OsdUseCustomColors" >
               <property name="toolTip" >
                <string>Check to enable custom colors for the On-Screen-Display.</string>
               </property>
               <property name="whatsThis" >
                <string>Check to enable custom colors for the On-Screen-Display.</string>
               </property>
               <property name="text" >
                <string>Use &amp;custom colors</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" >
               <item>
                <spacer>
                 <property name="orientation" >
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType" >
                  <enum>QSizePolicy::Expanding</enum>
                 </property>
                 <property name="sizeHint" >
                  <size>
                   <width>16</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="Q3GroupBox" name="colorsBox" >
                 <property name="enabled" >
                  <bool>false</bool>
                 </property>
                 <property name="title" >
                  <string/>
                 </property>
                 <property name="flat" >
                  <bool>false</bool>
                 </property>
                 <property name="frameShape" >
                  <enum>Q3GroupBox::NoFrame</enum>
                 </property>
                 <layout class="QGridLayout" >
                  <property name="leftMargin" >
                   <number>0</number>
                  </property>
                  <property name="topMargin" >
                   <number>0</number>
                  </property>
                  <property name="rightMargin" >
                   <number>0</number>
                  </property>
                  <property name="bottomMargin" >
                   <number>0</number>
                  </property>
                  <item row="1" column="0" >
                   <widget class="QLabel" name="textLabel2_2" >
                    <property name="sizePolicy" >
                     <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
                      <horstretch>1</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="whatsThis" >
                     <string>The color of the OSD background.</string>
                    </property>
                    <property name="frameShape" >
                     <enum>QFrame::NoFrame</enum>
                    </property>
                    <property name="text" >
                     <string>Background color:</string>
                    </property>
                    <property name="wordWrap" >
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1" >
                   <widget class="KColorButton" name="kcfg_OsdTextColor" >
                    <property name="sizePolicy" >
                     <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                      <horstretch>2</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="toolTip" >
                     <string>Click to select the color of the OSD text.</string>
                    </property>
                    <property name="whatsThis" >
                     <string>The color of the OSD text.</string>
                    </property>
                    <property name="text" >
                     <string/>
                    </property>
                    <property name="color" >
                     <color>
                      <red>255</red>
                      <green>0</green>
                      <blue>0</blue>
                     </color>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1" >
                   <widget class="KColorButton" name="kcfg_OsdBackgroundColor" >
                    <property name="sizePolicy" >
                     <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                      <horstretch>2</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="toolTip" >
                     <string>Click to select the background color of the OSD.</string>
                    </property>
                    <property name="whatsThis" >
                     <string>The color of the OSD background.</string>
                    </property>
                    <property name="text" >
                     <string/>
                    </property>
                    <property name="color" >
                     <color>
                      <red>255</red>
                      <green>0</green>
                      <blue>0</blue>
                     </color>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0" >
                   <widget class="QLabel" name="textLabel2" >
                    <property name="sizePolicy" >
                     <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
                      <horstretch>1</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="whatsThis" >
                     <string>The color of the OSD text.</string>
                    </property>
                    <property name="text" >
                     <string>Text color:</string>
                    </property>
                    <property name="wordWrap" >
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="kcfg_OsdUseFakeTranslucency" >
             <property name="text" >
              <string>Make the &amp;background translucent</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Q3GroupBox" name="osdText" >
          <property name="title" >
           <string>Display &amp;Text</string>
          </property>
          <property name="flat" >
           <bool>false</bool>
          </property>
          <property name="frameShape" >
           <enum>Q3GroupBox::GroupBoxPanel</enum>
          </property>
          <layout class="QVBoxLayout" >
           <item>
            <widget class="QCheckBox" name="kcfg_OsdUsePlaylistColumns" >
             <property name="text" >
              <string>Display the same information as the columns in the playlist</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KTextEdit" name="kcfg_OsdText" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="lineWidth" >
              <number>2</number>
             </property>
             <property name="tabChangesFocus" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Q3GroupBox" name="groupBox22" >
          <property name="title" >
           <string/>
          </property>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QLabel" name="textLabel1" >
             <property name="text" >
              <string>&amp;Duration:</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
             <property name="buddy" >
              <cstring>kcfg_OsdDuration</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QSpinBox" name="kcfg_OsdDuration" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
               <horstretch>2</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip" >
              <string>The time in milliseconds to show the OSD. The value must be between 500 ms and 10000 ms.</string>
             </property>
             <property name="whatsThis" >
              <string>The time in milliseconds to show the OSD. The value must be between 500 ms and 10000 ms.</string>
             </property>
             <property name="buttonSymbols" >
              <enum>QAbstractSpinBox::UpDownArrows</enum>
             </property>
             <property name="specialValueText" >
              <string>Forever</string>
             </property>
             <property name="suffix" >
              <string> ms</string>
             </property>
             <property name="minimum" >
              <number>0</number>
             </property>
             <property name="maximum" >
              <number>600000</number>
             </property>
             <property name="singleStep" >
              <number>1000</number>
             </property>
             <property name="value" >
              <number>5000</number>
             </property>
            </widget>
           </item>
           <item row="0" column="3" >
            <widget class="KComboBox" name="kcfg_OsdScreen" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
               <horstretch>2</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip" >
              <string>The screen that should display the OSD.</string>
             </property>
             <property name="whatsThis" >
              <string>The screen that should display the OSD.</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" >
            <widget class="QLabel" name="textLabel2_3" >
             <property name="text" >
              <string>Sc&amp;reen:</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
             <property name="buddy" >
              <cstring>kcfg_OsdScreen</cstring>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >osd.h</include>
  <include location="local" >ktextedit.h</include>
  <include location="local" >knuminput.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_OsdUseCustomColors</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorsBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_OsdEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>mainBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_OsdUseCustomColors</sender>
   <signal>toggled(bool)</signal>
   <receiver>OsdConfig</receiver>
   <slot>useCustomColorsToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_OsdUsePlaylistColumns</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_OsdText</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
